satpy/tests/reader_tests/test_gld360_ualf2.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Unittesting the GLD360 ualf2 reader."""
+
+import tempfile
+import unittest
+
+import numpy as np
+
+from satpy.readers.gld360_ualf2 import Gld360Ualf2FileHandler
+from satpy.tests.utils import make_dataid
+
+
+class TestVaisalaGLD360TextFileHandler(unittest.TestCase):
```

Comment:
class name here should reflect the new `FileHandler`-name, change to `TestVaisalaGld360Ualf2FileHandler`